Makefile

```diff
@@ -54,6 +60,34 @@ help:
 	@echo "  QA_CONTAINER"
 	@echo "             tag for the qa docker container [current: $(QA_CONTAINER)]"
 
+docker-qa: export TEST_OPTS := $(TEST_OPTS)
+docker-qa: export GCOVR_ISOLATED_TEST := zkQEVaBpXF1i
+
+ifeq ($(CC),all)
+
+setup-dev:
+	$(RUN_ALL_COMPILERS)
+
+qa:
+	$(RUN_ALL_COMPILERS)
+
+lint:
+	$(RUN_ALL_COMPILERS)
+
+check-format:
+	$(RUN_ALL_COMPILERS)
+
+test:
+	$(RUN_ALL_COMPILERS)
+
+doc:
+	$(RUN_ALL_COMPILERS)
+
+docker-qa:
+	$(RUN_ALL_COMPILERS)
+
+else
```

Comment:
Is it really meaningful to run targets other than qa/docker-qa for each compiler version? Extracting the `RUN_ALL_COMPILERS` pattern seems sensible, but I'd only apply it to those two targets.

Reply:
Done.